### ISSUE TITLE
CH 31 | Change Directional Language

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#68022]
## Issue Description
_CAIA issue summary #64163. Consider tackling this issue in tandem with #68027._

### Staging or Production Link
- [Staging Link](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)

### Prototype Link
- [Mural User Flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1684161597392/5c1f8869cf5d03b792c8b82cf19b39dc26c486c2?sender=u0e8ac1d6d7681bb7e1b80558)

### Test Users
- vets.gov.user+24@gmail.com
- [staging test user repository](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-users.csv)

### Details
While it's not in the content style guide yet, CAIA recommends that you don't use directional language (above, below, left, right). Assistive technology users navigate through a web page in a variety of ways, such as by headings, links, and landmarks, and may tab backwards through a page. Directional language like this can be confusing. 

![Heading with 'Follow the steps below' language](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/65ae175b-ccb2-490c-9a09-623b97fd3983)

### Reproduction steps
1. Go to [the form](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/introduction)
2. Scroll down to this heading: "Follow the steps below to apply for Veteran Readiness and Employment benefits."

### References, articles, or WCAG support
Google's [Write accessible documentation](https://developers.google.com/style/accessibility) guide - see "Don't use directional language to orient the reader"

---
## Tasks
- [ ] Change heading to "Follow these steps to apply to apply for Veteran Readiness and Employment Benefits."

## Acceptance Criteria
- [ ] Heading has been updated.
- [ ] Designer signs off in staging environment.
---

**Comment by tblackwe** [comment]
@fiorella-io @juliepedtke Can you please review the suggested copy and verify or suggest an alternative?

**Comment by fiorella-io** [comment]
@tblackwe This sounds good and it implicates all forms we'll be enhancing.
